openke_testing.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+for i in {1..5}
+do
+  traintimes=$((500*$i))
+  learning_rate=$(expr 0.001*$i | bc)
+  number_batches=$((50*$i))
+
```

Comment:
Dom - this seems pretty straightforward, though I did have to change the imports and file locations to get this to work. 

For instance, I think my `PYTHONPATH` variable is different, and it's making the imports not work here. If I change the imports so that they start with `multivac.` - (e.g., `multivac.src.rdf_graph.map_queries`) then it works. Likewise, I don't have a `glove` directory in my local repo, whereas I do have a `models` directory where the GloVe embeddings are stored. We should be explicit in the script and/or the commit message or comments explaining those kind of requirements.